tests/playbooks/kubevirt_pvc.yml

```diff
@@ -3,6 +3,11 @@
   hosts: localhost
   connection: local
   tasks:
+    - name: Check for known open issues
```

Comment:
Should we add this to every tests playbook? I see that we defined more variables in `known_issues.yaml`

Reply:
I wanted to make it run only once globally, but ansible doesn't allow that. So it needs to be included at the beginning of each playbook. It made sense to me to only include it in playbooks that are known to have open issues, but I guess it doesn't really matter if it runs for every playbook, it's just one http call. I'll put it everywhere.

Reply:
I would added it everywhere so we can easily track issues related to other modules.